src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.props

```diff
@@ -9,11 +9,12 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
 Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
-<Project ToolsVersion="14.0" TreatAsLocalProperty="RuntimeIdentifier">
+<Project ToolsVersion="14.0" TreatAsLocalProperty="RuntimeIdentifier;RuntimeIdentifiers">
   <PropertyGroup>
     <OutputType>exe</OutputType>
 
     <RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
+    <RuntimeIdentifiers>$(RuntimeIdentifier)</RuntimeIdentifiers>
```

Comment:
I'm unsure if we should just unset the property entirely or set it to `$(RuntimeIdentifier)` here. Did you try it both ways? Were there any behavioral differences?

Reply:
I would set this explicitly to `browser-wasm`